libraries/classes/Controllers/Database/StructureController.php

```diff
@@ -393,20 +398,21 @@ protected function displayTableList(): string
                     $current_table['Collation']
                 );
                 if ($tableCollation !== null) {
-                    $collationDefinition = '<dfn title="'
-                        . $tableCollation->getDescription() . '">'
-                        . $tableCollation->getName() . '</dfn>';
+                    $collationDefinition = $this->template->render('database/structure/collation_definition', [
+                        'valueTitle' => $tableCollation->getDescription(),
+                        'value' => $tableCollation->getName(),
+                    ]);
                 }
             }
 
             if ($this->isShowStats) {
                 $overhead = '-';
                 if ($formatted_overhead != '') {
-                    $overhead = '<a href="' . Url::getFromRoute('/table/structure', $tableUrlParams)
-                        . '#showusage">'
-                        . '<span>' . $formatted_overhead . '</span>&nbsp;'
-                        . '<span class="unit">' . $overhead_unit . '</span>'
-                        . '</a>' . "\n";
+                    $overhead = $this->template->render('database/structure/overhead', [
+                        'url' => Url::getFromRoute('/table/structure', $tableUrlParams) . '#showusage',
```

Comment:
I think it's better to call the function inside the template:
```suggestion
                        'table_url_params' => $tableUrlParams,
```